### DISCUSSION TITLE
Caanot set the FulentProvider's theme from the context

### DISCUSSION THREAD

**Discussion by fauberon** [microsoft/fluentui#31791] [Q&A]
Using react 18.3.1, "@fluentui/react-components" : "9.54.2"; 
I set FluentProvider at the root element with a "theme" variable used in the context;
When I try setting the context to one of the standard value, such as webDarkTheme, error raises:
*************
Uncaught Runtime error 
Objects are not valid as a React child (found: object with keys {borderRadiusNone, borderRadiusSmall, etc
*************
Any idea? Is this a known limitation of FluentProvider, and shall I use https://react.fluentui.dev/?path=/docs/utilities-theme-createcssrulefromtheme--default (there is an exmple of theme switching).
Thanks for your help.

**Comment by khmakoto** [comment]
@fauberon do you have a minimal repro in something like Stackblitz that I could take a look at?
